modules/express/src/lightning/README.md

```diff
@@ -0,0 +1,220 @@
+  enterprise,
+  passcodeEncryptionCode
+}
+```
+
+* `label`: the name of the wallet
+* `passphrase`: This is the passphrase that encrypts the keys on the wallet.
+* `enterprise`: the ID of the enterprise you want to create the wallet under
+* `passcodeEncryptionCode`: This is the passphrase that encrypts the walletPassphrase. This is used in recovery scenarios
+if you lost the `passphrase`, where the encrypted backup is stored in BitGo's databases. This should be stored in a safe location.
+
+This will start the process of creating a wallet. Once the watch-only lightning node is created, you will receive an email for the next step.
+
+#### Step (2) Create the `lightningSignerConfig.json`
+
+Once we have deployed a `signer lnd` node and created the wallet, we have the ability to
+create the `lightningSignerConfig.json`, which gives the relevant information for `Express`
+to talk to the `signer lnd` node:
+
+```
+{
+  walletId: { url, tlsCert }
+}
+```
+
+* `walletId`: This is the `id` for the wallet that was created in step 1.
+* `host`: This is the url/hostname for how `Express` can contact the `lnd signer` node via REST.
+If they are deployed in the same namespace, you can use `0.0.0.0`. Note that you need to include the port (LND default for REST is 8080).
+* `tlsCert`: This is the TLS certificate created by the `signer lnd` node encoded
+to base64. You can generate get this by deploying the `signer lnd` node and it will create
+one automatically if you do not provide one.
+
+When deploying the `Express` instance, pass in the config using the flag `--lightingSignerConfig lightingSignerConfig.json`. __This will require restarting the `Express` server so it can run with this flag.__
+
+You can test your connection to `Express` and the `signer lnd` node (via `Express`) with the
+following endpoints:
+
+```
+GET /api/v2/ping (ping Express)
+GET /api/v2/[t]lnbtc/wallet/:walletId/state (ping signer lnd via Express)
+```
+
+Both should return `200`. If `/api/v2/ping` is not working, you cannot correct connect to Express. If `/api/v2/[t]lnbtc/wallet/:walletId/state` is not working, then `Express` cannot properly connect to the the `signer lnd`.
+
+#### Step (3) Initialize the `signer lnd` node
+
+Once we are able to communicate with the `signer lnd` node from Express, we can initialize the `signer lnd` node with the following endpoint:
+
+```
+POST /api/v2/[t]lnbtc/wallet/:walletId/initwallet
+{
+  passphrase: string
+}
+```
+
+This will initialize the lnd using the [`initwallet`](https://lightning.engineering/api-docs/api/lnd/wallet-unlocker/init-wallet/) LND route. This will create the admin macaroon for the signer node and an encrypted version (to the wallet passphrase) will be uploaded to the wallet.
+
+#### Step (4) Update Wallet with relevant information
+
+Once we have a deployment for `Express` and `lnd signer` node, we can update the wallet:
+
+```
+PUT /api/v2/[t]lnbtc/wallet/:walletId
+{
+  signerHost: string,
+  tlsCert: string,
```

Comment:
Ideally this would come from the config itself and the customer wouldnt have to provide that here again